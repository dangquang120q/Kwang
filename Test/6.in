20
78
6015332 3697474 9108204 7211771 3181212 4835231 1781632 3689080 1078345 3468202 1891398 5147830 1607730 1662695 624000 7112689 4898555 1948991 2833673 5487268 3268647 9786741 1037191 7454817 2125293 9699451 7161553 8392720 2161115 6393996 4979332 7865117 9706595 9231121 2021675 7856892 4037584 9266833 8220702 5545288 8185105 2634294 8551919 957206 6782460 9605574 5082333 7256876 8763852 6363669 1163795 4436029 4975674 540598 8871673 1637196 9518462 9962463 4949097 8678491 7559373 6166768 996056 5889410 2067660 3101255 1098117 4860033 626128 603892 4638524 2133141 637745 5443799 7245919 947789 6397855 8613090 
6
2405692 9615005 7802435 913094 5767424 8153530 
38
4442555 7442365 4201120 5876764 1623034 168896 4370893 1401143 175483 2607551 6015783 4027320 682979 7616246 7987383 1855967 1180485 9717787 4844479 4126726 4121317 1630662 5611143 6442356 8323890 8171808 3644099 593638 4144335 8706210 6053089 2492837 8569442 3040107 7720154 8250728 1942940 6003198 
82
8297112 9114402 8797232 7863467 4698257 1783325 8363364 9606510 9345693 1823610 5992620 3582416 511576 1804498 5398533 5210272 5906111 7964808 363364 5596189 3165270 9149371 4601817 8432985 9190277 5969359 457191 7801768 9204816 9261069 4474003 1452370 2485794 1388441 4484250 854702 9291591 2089833 3473726 1007020 6844258 6729837 1896003 1621292 6987432 2479202 6644523 2490145 1484228 6763449 8714070 2182502 9923457 5104382 546906 9029857 8507047 6158572 2485720 2742149 855285 9835123 1824657 8316799 7487472 4562328 8561325 224905 8859558 3463106 1721343 432035 2457751 7914906 2497497 312302 3218374 7904462 6241764 8716796 3750397 8490559 
92
5836034 169027 4234411 6479894 8690176 7785361 2368578 4047647 2906090 3847376 2113912 925620 8057180 8969381 8559631 6253705 9152080 7547542 9765043 1483417 6571028 6260475 3020282 6530904 2070075 4755000 940060 3880079 7693540 8053260 8490716 8067812 7985494 4049134 3886372 859346 8025724 6701109 7864290 4186181 6271675 7891437 9118590 4016418 2918848 6684108 724632 2695872 1166211 7194475 9139226 5910031 4903672 6460674 3338841 5548839 5162540 1638869 8954378 8279564 1738780 7153768 9161553 978501 8671669 8806091 7255809 2519257 3133982 4848742 532758 644971 9767653 8497419 6848336 5028166 5496499 8259330 1367312 6349930 4171583 8750416 5938336 5838176 9241627 8329554 9677158 9542095 8801253 2762623 9622132 354897 
65
3476791 5994935 5180779 6146873 7788072 9974003 3774899 5751491 2246880 9197641 9322601 9657325 576368 8493668 7909170 5155182 4149848 7232760 219618 3409769 8176540 9465184 6654563 7760168 3385696 5233040 8502121 8068620 7051492 9834024 1894891 841732 6221874 3768443 9653785 8598977 7403057 8671445 9953950 8715084 2575062 3862258 5693914 5576725 7258086 3400774 8910284 7126720 4548726 5501345 2561292 600155 5337847 8848353 2538392 7159987 7968838 829374 7695319 6136835 1304567 3131511 5979021 2849232 7481580 
18
9505166 9724399 9330096 8738552 4523311 1004260 900572 3356465 711696 2071156 7793448 4397906 5968205 9538550 8923753 8678756 9408095 3681681 
17
7145282 8533473 8020784 3389146 1260973 1894574 6237792 340825 4866911 1768976 8568937 8538350 2804006 8211032 2648962 8392039 6775140 
48
5837378 2572620 4326529 344996 3094558 181555 895202 9796080 8030444 492072 669313 1015475 2799815 5583832 386201 4680474 5294580 4600852 9453627 3372343 495233 9846851 7780232 7762840 7710487 1721335 1561797 5315407 5721726 7950200 9624920 8350414 7465504 4262065 7234522 4934987 4250084 2934130 2854802 1291735 7581649 2436342 3192272 1296595 9344452 6397806 2960257 3066308 
18
5439238 7116755 9411428 4948999 2730508 9441635 9651566 9720864 3125185 7564614 4908302 2538703 5458252 7542824 968239 8460516 8487281 1673214 
77
6196276 4525337 2301493 7048287 426149 9368438 2735714 8822905 8436772 2999718 9108780 1409957 7238161 2378463 7088078 5300499 5083090 2674142 4208093 7719656 6114886 132844 1250492 5179233 3892424 3849322 7266463 8830846 7194484 8812667 9899870 3173239 4919732 4547983 9995821 2189446 5452983 4339217 4941609 1475223 4915749 5982583 8228988 1157232 5327518 5701689 1195862 952559 8432318 4398211 2073390 9814093 9840034 9794717 9216909 6158496 2183616 2783313 9192365 7732555 6675298 4737997 7162649 6563487 7424196 1465528 4881946 8869100 1032353 2424816 1293293 7364003 9404503 3679739 1596194 1796085 1909795 
80
605707 8061655 5842751 2368988 8639768 6767221 2462390 864090 8603004 7534733 8719686 7218640 8346467 7915662 5273564 9027671 5583358 2516544 5563137 7930029 5750848 8335237 2139227 4062521 7386482 2499689 8619159 3569831 2241788 5282339 3507065 4749516 4286082 2728698 4883113 2629522 9917513 2907307 144347 3922088 1502138 5817450 2211466 8238338 5964204 1019634 6085718 5896671 7810850 1949629 3322398 5610053 4923530 3547885 7021837 7975320 8116946 7946180 9147393 626299 1433463 3203051 7963638 2789866 2293408 7631839 1157536 5885932 6002711 8938057 7568642 2700018 8621157 7115046 1653266 4675936 8866689 6425482 9696116 5947332 
26
5874515 822282 5653406 3051842 4710003 7145659 6372016 6218044 9373017 4719553 8151205 464132 6533630 7888165 6511903 6360317 9831998 6824973 5615072 828326 1005010 6559431 9910333 5725726 5088701 9950467 
53
1184739 525410 8949999 6426616 5408186 2312022 5251509 2624066 192726 9649394 245802 5377849 1450313 1858525 6537474 2880206 5221830 774392 9298427 4831652 3504683 8897889 2638395 7425419 4280421 184559 2530251 8756386 6165945 9015385 3427549 5960829 4046434 2370732 9942320 6747636 6300301 506791 9819012 4185862 5934241 6254926 2726772 2136010 1509952 6965891 9374848 3935020 4789358 2082431 3185510 8137189 3794839 
89
5392155 1202478 1531864 6982632 137972 5147780 3083984 5701642 8419815 7919208 6606490 146082 8337975 4812797 5516791 3793725 7824077 1257404 3558111 5804341 6374847 981065 317112 3596608 8088978 5700223 4787502 8897526 1102095 8169908 8218687 3393109 4203849 6053457 6482113 9200164 5991004 9690299 8513584 9323483 5507157 9519743 3377496 6883948 970460 701096 8596487 2682868 4022681 1553158 7784849 227883 1234690 7730344 6557376 638763 6818609 7414344 4891059 4534684 3542552 8899534 3879570 6816214 8783626 8543849 4233945 4005287 8957322 6023756 1570606 3804554 5157733 9898459 8133579 7642841 8563152 7488760 2757399 1898477 2498837 8837008 6898400 9710219 4802122 9774229 5285327 8984946 5746360 
89
7762516 4287607 3453813 7028787 7371807 380183 541386 6161721 8009799 7106486 5548914 9655806 626065 8062790 3848266 7046499 4530479 7777177 2777848 6984974 1364678 6864930 8012592 3651021 4185395 3472005 4393851 1714420 7708614 1423602 3392084 9586413 3496422 1571680 9780394 6787426 582392 8085236 5504299 360615 3654788 5693530 6406237 7394522 9225586 9476687 9186353 7427213 8915311 3234619 9830654 4530324 1439013 3474002 6980085 7394669 9188116 5291338 223239 5854286 5177910 6543518 9335436 1931103 6239583 4140142 8455429 5678556 8280961 4146257 8527753 4234082 1828171 5223707 8672792 7231892 2917455 3093265 3807021 4949517 8379239 7250641 5156515 9505223 4367232 8851246 3558934 4852663 1380661 
34
7917106 2095273 361088 6046919 8898675 2604921 790729 7296463 1177368 8050209 2839214 4206070 1180165 6321604 999627 1007992 3815862 8244104 1647917 4370386 5945104 2830743 534794 3537993 9583849 5448266 3118286 519223 1069130 4064294 6627058 9881353 6717256 5507028 
57
5409217 1874706 7968764 228517 9167502 4493625 7352188 5629103 8976042 6300600 3636483 5236029 4838297 1854331 1241086 5642881 2686820 493050 110200 4877334 3824798 315005 9265245 8139522 3123928 6497095 6591532 3636612 5403398 6798361 3127929 5614090 8370042 8116174 1194252 346486 1998379 7388250 3617328 7634706 7677403 4886439 4705475 3983134 7523203 7421409 6753378 1262138 4177677 2375643 1424723 708114 557828 5427567 1118661 1481150 2381013 
75
636889 6211108 2688115 9455778 3378267 6326492 965754 3387986 8730491 5602076 8366056 505115 6916434 6155846 6973058 3216842 6889002 2145733 2489964 7005652 9748018 7556993 6246780 269518 4071794 8392508 5157924 3449644 2812938 9444568 8601030 6043641 9356361 125232 6303797 8276301 8220845 3811593 4735921 9034418 4152846 4937445 7485350 7905537 8782279 7877368 5851689 1567086 9069048 6590897 639223 7087698 621084 4012484 5925037 326335 5032072 6425070 6965909 9132654 1881572 2931715 9709998 5319310 2736074 406733 6772938 4870481 6007110 8451365 4157387 1556958 9115899 9508388 7926317 
51
4773160 792993 2023579 8474446 9983366 7168636 5726693 3618484 5700639 3686194 605223 5825673 9342798 5979758 2554740 7866554 768365 9824237 3780007 647760 8674025 2874959 8716310 8244374 4371485 6431075 8116233 5614960 6616028 3034282 1859526 1289745 6117670 5609510 2427861 7134078 8380464 9652847 5667136 6548835 778877 3068766 9673170 6197233 4669042 5325243 744490 8237844 8753986 2334893 7284334 
