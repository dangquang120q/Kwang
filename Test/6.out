29455799
25570970
26993439
29365090
29209854
29761977
28768115
25640760
29267851
28814065
29735725
28761022
29692798
29410726
29382907
29266854
28363877
27408789
28966404
29480773
